source/blog/2018-04-01-the-emberjs-times-issue-43.md

```diff
@@ -63,19 +87,27 @@ and give your thoughts and suggestions in the discussion.
 
 <div class="blog-row">
   <img class="float-right small transparent padded" alt="Office Hours Tomster Mascot" title="Readers' Questions" src="/images/tomsters/officehours.png" />
-  <p>In this week's brand new Readers’ Question it's all about the new and cool and shiny that will 
-  be coming to your Ember apps soon: When will we be able to make use of the decorator syntax when writing Ember apps? 
+  <p>In this week's brand new Readers’ Question it's all about the new and cool and shiny that will
+  be coming to your Ember apps soon: When will we be able to make use of the decorator syntax when writing Ember apps?
   </p>
  <p>In his detailed write-up <a href="https://github.com/pzuraq">@pzuraq</a> will answer everything you need to know about the current state of the decorators
-  proposal itself and when you will be able to decorate your own application. Read the full answer 
-  on the <a href="#">offical Ember forum</a>.
+  proposal itself and when you will be able to decorate your own application. Read the full answer
+  on the <a href="#">official Ember forum</a>.
 </div>
-<a class="ember-button ember-button--centered href="#">Read more</a> 
+<a class="ember-button ember-button--centered href="#">Read more</a>
 
 ---
 
+## Slack Stars
+Ember Times would like to give a shoutout to <a href="https://github.com/runspired">@runspired</a> (Chris Thoburn), for being incredibly helpful and responsive in a majority of the Slack channels this last week. Chris is major contributor to the ember ecosystem and an integral part of the ember-data subteam.  He has been super supportive in helping community members migrate through the many versions of Ember data from 2.12 all the way to 3.2.0-beta.2.  Additionally Chris shares his intimate knowledge of the ember-data internals to make it easier for everyone to understand why things work in a particular way and to also shed some light on how ember-data will work in the future.  Thanks @runspired!
+
+## We love Ember Times
+Do you? If so, contribute!  Ember Times is always looking for people to help spread the word about all things Ember.  Drop by the Slack channel to say "hi!" and learn how to get started.        
```

Comment:
It would be really cool to add a link to the `#embertimes` channel here, too - can you add it in the text and in the header title (if you think it's suitable)?